### PR TITLE
WIP: add setup script

### PR DESCRIPTION
**Pull request by CJ-Wright**
Helper for those projects which haven't been graced by rever yet and would like to get started quickly.

### CODE REVIEW

File: rever/main.xsh [hunk 1]
```diff
@@ -159,5 +184,20 @@ def main(args=None):
         env_main(args=args)
 
 
+def setup(args=None):
```
Comment (scopatz) [inline rever/main.xsh]: I think a better model is for each activity to have a setup or bootstrap method.  Then in the main file, yo just dispatch to the activities setup method

Comment (CJ-Wright) [reply rever/main.xsh]: Yes that is a good idea.

File: rever/main.xsh [hunk 2]
```diff
@@ -10,6 +11,30 @@ from rever import environ
 from rever.dag import find_path
 
 
+NEWS_TEMPLATE = '''
```
Comment (scopatz) [inline rever/main.xsh]: This doesn't need to still be here does it?

File: scripts/setup [hunk 3]
```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env python3 -u
+from rever.main import setup
+setup()
```
Comment (scopatz) [inline scripts/setup]: I don't think there should be a top-level setup script. Rather the normal rever command should have a setup argument

Comment (CJ-Wright) [reply scripts/setup]: How do we trigger the setup?

File: rever/main.xsh (hunk 2 above)
Comment (CJ-Wright) [reply rever/main.xsh]: Yes sorry

File: scripts/setup (hunk 3 above)
Comment (CJ-Wright) [reply scripts/setup]: Should there be a -s flag?

Comment (scopatz) [reply scripts/setup]: Yeah, -s or --setup

**Pull request merged**